### PR TITLE
update battle-stat,statLevelChangDescription for translate to edit

### PR DESCRIPTION
**Pull request by Rei1mu**
update battle-stat for zh-CN. en. 
battle-stat,and battleStatLevelChangeDescription can be edit for translate at 'src/locales/xx/battle-stat.ts'

### CODE REVIEW

File: src/phases.ts [hunk 1]
```diff
@@ -2927,7 +2927,7 @@ export class StatChangePhase extends PokemonPhase {
           : `${relLevelStats.slice(0, -1).map(s => getBattleStatName(s)).join(', ')}${relLevelStats.length > 2 ? ',' : ''} and ${getBattleStatName(relLevelStats[relLevelStats.length - 1])}`;
       } else
         statsFragment = getBattleStatName(relLevelStats[0]);
-      messages.push(getPokemonMessage(this.getPokemon(), `'s ${statsFragment} ${getBattleStatLevelChangeDescription(Math.abs(parseInt(rl)), levels >= 1)}!`));
+        messages.push(getPokemonMessage(this.getPokemon(), i18next.t('battleStat:levelChangeDescripion', { statsFragment: i18next.t(`battleStat:${statsFragment.replace(/\./g, '').replace(/[ ]/g, '_')}`), description: i18next.t(`battleStat:${getBattleStatLevelChangeDescription(Math.abs(parseInt(rl)), levels >= 1).replace(/\./g, '').replace(/[ ]/g, '_')}`) })));
```
Comment (bennybroseph) [inline src/phases.ts:2930]: what do the `.replace()` calls do here?

**Review by bennybroseph — APPROVED** [review]

**Review by bennybroseph — CHANGES_REQUESTED** [review]
what do the .replace() calls do here?

File: src/phases.ts (hunk 1 above)
Comment (bennybroseph) [inline src/phases.ts:2930]: ones like this for reference

**Review by bennybroseph — CHANGES_REQUESTED** [review]

**Comment by Adri1** [comment]
Isn't the "!" fragment missing or I misslooked?

**Comment by Rei1mu** [comment]
> Isn't the "!" fragment missing or I misslooked?
> 
> EDIT : It is possible to give to gendered languages the possibility to choose a determiner? Because in French for example before the stat name, you have to put a mandatory determiner like "the" but is either masculine/feminine/genderless Example :
> 
> * "[the (feminine)] Defense"
> * "[the (genderless, without any space between the derterminer and the word)] Attack"
> * "[the (masculine)] critical-hit ratio"

"!" maybe i  missed , but it can edit on 
 ```
"levelChangeDescripion": "'s {{statsFragment}} {{description}}", 
 "levelChangeDescripion": "'s {{statsFragment}} {{description}}!",
```
about [the] 's problem, this is just part of that . 
i think it means will change the 'the'  at getPokemonMessage() ?

```
export function getPokemonMessage(pokemon: Pokemon, content: string): string {
  return `${getPokemonPrefix(pokemon)}${pokemon.name}${content}`;
}
export function getPokemonMessage(pokemon: Pokemon, content: string): string {
  return `The ${getPokemonPrefix(pokemon)}${pokemon.name}${content}`;
}
```
like this ?  sorry i'm not good at languages

**Comment by Rei1mu** [comment]
> what do the .replace() calls do here?

It's update. please check

**Comment by bennybroseph** [comment]
Please address the above concerns

**Comment by Tempo-anon** [comment]
Closing this PR as it's a bit out of date. If this is still required please submit a new PR

**Pull request closed without merging**